.changesets/exp_pencil_hunk_tornado_backyard.md

```diff
@@ -0,0 +1,7 @@
+### Extended references in Apollo usage reports ([Issue #ROUTER-360](https://apollographql.atlassian.net/browse/ROUTER-360))
```

Comment:
Subtle and I've fixed via https://github.com/apollographql/router/pull/5610 but for future review awareness, let's not include the Jira links — they're not useful for the community.  Instead, use our PR numbers in the absence of the GitHub Issue.  cc @bonnici @BrynCooke @bnjjj